### ISSUE TITLE
Result of translating using an array in the options hash has changed

### ISSUE THREAD

**Issue by joshuaswift** [ember-intl/ember-intl#1313]
- [x] I am on the [latest](https://github.com/ember-intl/ember-intl/releases/latest) ember-intl version
- [x] I have searched the [issues](https://github.com/ember-intl/ember-intl/issues) of this repo and believe that this is not a duplicate

### Environment

- Ember Version: 2.18.0
- Ember CLI Version: 2.18.2
- Ember Intl Version: 5.1.1
- Browser(s): Chrome
- Node Version: 12.4.0

Since upgrading from 4.3.2 to 5.1.1, if we pass an array as an option to the translation the`format-message` internal formatter adds a comma before and after the array, see below.

**Before upgrade:** 
```
The Receipts with id: 1 could not be deleted. Please, try again. If the problem persists, please contact support for further information.
```

**After upgrade:**
```
The Receipts with id: ,1, could not be deleted. Please, try again. If the problem persists, please contact support for further information.
```

I noticed the commas get added in the [following line](https://github.com/ember-intl/ember-intl/blob/master/addon/-private/formatters/format-message.js#L73) if that helps diagnose the issue.

### Steps to Reproduce

1. Pass array as an option to the translation like below.`rejectedIds` is an array of strings currently, but I've also tried passing as an array of Ints and we get the same issue.

```
"goods_in_detailed_delete": "The Receipts with id: {rejectedIds} could not be deleted. Please, try again. If the problem persists, please contact support for further information.",
```

The service is called like so:

```
intl.t('inbound_goods.notifications.goods_in_detailed_delete', {
   rejectedIds,
 }),
```

Let me know if you have any questions 😄

**Comment by jasonmit** [comment]
Unsure if providing an array as an argument to a translation was ever "officially" supported in [`intl-messageformat`](https://github.com/formatjs/formatjs/tree/5b218606c3ec03e332f472706fffd04153c20dae/packages/intl-messageformat).  I'll need to investigate more, a quick grep hasn't yielded anything.

**Comment by joshuaswift** [comment]
Thanks for the quick reply, this solution will work for now 👍

**Comment by sandstrom** [comment]
@joshuaswift I've noticed something similar with SafeString instances, but in that case we've addressed it by adding SafeString support (https://github.com/ember-intl/ember-intl/pull/1294).

I think it's `intl-messageformat` that does this conversion (https://github.com/formatjs/formatjs/tree/master/packages/intl-messageformat), so it's outside the scope of this library.

However, I strongly agree with @jasonmit that this is has never been supported officially and if it did work in your case earlier that was a bug, not by design. 

The correct solution is to do the join before passing in the argument to `intl.t`, like jason illustrated. Also, list formatting varies per language, so you'd want to use something like [this lib](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat) anyway, and that's yet another reason for formatting your array into a list *before* passing it to `intl.t`.
